packages/select-rich/test/lion-select-rich-dialog-integration.test.js

```diff
@@ -0,0 +1,53 @@
+import { OverlayMixin } from '@lion/overlays';
+import { LitElement } from 'lit-element';
+import { defineCE, fixture, html, expect, unsafeStatic } from '@open-wc/testing';
+import '../lion-select-rich.js';
+import '../lion-options.js';
+import '../lion-option.js';
+
+const tagString = defineCE(
+  class extends OverlayMixin(LitElement) {
+    render() {
+      return html`
+        <button slot="invoker">invoker button</button>
+        <slot name="_overlay-shadow-outlet"></slot>
+        <div id="overlay-content-node-wrapper">
+          <div slot="content">content of the overlay</div>
+        </div>
+      `;
+    }
+  },
+);
+const tag = unsafeStatic(tagString);
+
+describe('Select Rich Integration tests', () => {
+  it('works inside a dialog', async () => {
```

Comment:
Should we do some more smoke testing, e.g. open the dialog and assert it, open the rich select and assert it, select an option and assert the modelValue? Now we just check that no error is thrown (good start though).

Reply:
Good be a nice idea for the future indeed.